src/browser/api_protocol/transport_strategy/ack.ts

```diff
@@ -5,7 +5,29 @@ export class AckMessage {
     public readonly action: string = 'ack';
     public breadcrumbs?: Array<any>;
     public correlationId: number;
+    public readonly originalAction?: string;
     public payload: AckPayload | NackPayload;
+
+    constructor(breadcrumbs?: Array<any>, originalAction?: string) {
+        if (breadcrumbs) {
+            this.breadcrumbs = breadcrumbs;
```

Comment:
Maybe better to initiate with given breadcrumbs or empty array, so that `addBreadcrumb` doesn't have to keep checking and can simply `push`

Reply:
The intent was to _only_ add the new array to every payload when breadcrumbs are present.